slp_base/tests/util/otm.py

```diff
@@ -6,13 +6,14 @@
 from slp_base.slp_base.otm_file_loader import OtmFileLoader
 from slp_base.slp_base.schema import Schema
 
-public_cloud_id = 'b61d6911-338d-46a8-9f39-8dcd24abfe91'
+public_cloud_id = '804b664a-7129-4a9e-a08c-16a99669f605'
```

Comment:
Here we have an public_cloud_id var for testing because it is the id expected of the default trust zone expected in the tests.
The other trust zone id expected depends on the source file, but our default trust zone is "Public Cloud" and his id is  a deterministic uuid built from his name "Public Cloud".
If any trust zone comes in the mapping file set as default, its id will be built from his name. The default "Public Cloud" will have always 804b664a-7129-4a9e-a08c-16a99669f605 as id.
The reason for do that is for having a consistent OTM (for upgrading an existing project, for example), with the components, trust zones and data flows always with the same ids if the source wasn't change

Reply:
Completely agree with you! I got the point now!
Nothing to do here then. Thanks for the explanation, Santi!